packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -478,6 +489,52 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
+  // Logic for updating `fieldTranslations`
+  // user has no description translations OR user is changing the description
+  const descriptionTranslationsNeeded =
+    eventType.fieldTranslations.filter((trans) => trans.field === EventTypeAutoTranslatedField.DESCRIPTION)
+      .length === 0 || newDescription;
+  const description = newDescription ?? eventType.description;
+
+  if (
+    ctx.user.organizationId &&
+    autoTranslateDescriptionEnabled &&
+    descriptionTranslationsNeeded &&
+    description
+  ) {
+    const userLocale = ctx.user.locale;
+
+    // Use process.nextTick to handle translations in the background
+    process.nextTick(async () => {
```

Comment:
This isn't the background processing we are looking for since this tRPC handler is handling an HTTP request.

We should do this in a `Tasker` task that is picked up by the cron system. It's how we defer longer-running tasks fully to the background.

Reply:
Addressed